lib/bundler/templates/newgem/Rakefile.tt

```diff
@@ -4,6 +4,8 @@ require "rake/testtask"
 
 Rake::TestTask.new(:test) do |t|
   t.libs << "test"
```

Comment:
`libs` is like `ruby -I`.
So we need to include the lib folder, and the test.
`test` is necessary because of `require test_helper`